external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -121,4 +134,167 @@ class KafkaDataConsumerSuite extends SharedSQLContext with PrivateMethodTester {
       threadpool.shutdown()
     }
   }
+
+  test("SPARK-25151 Handles multiple tasks in executor fetching same (topic, partition) pair") {
+    val topic = "topic" + Random.nextInt()
+    prepareTestTopicHavingTestMessages(topic)
+    val topicPartition = new TopicPartition(topic, 0)
+
+    import ConsumerConfig._
+    val kafkaParams = Map[String, Object](
```

Comment:
Maybe `kafkaParams` is common.

Reply:
Yeah same configuration seems to be used in multiple places. Will deduplicate.